tests/run-pass/current_dir.rs

```diff
@@ -1,4 +1,3 @@
-// ignore-windows: TODO the windows hook is not done yet
 // compile-flags: -Zmiri-disable-isolation
 use std::env;
 use std::path::Path;
```

Comment:
It might be a good idea to extend this test with a failing case, like
```rust
assert_eq!(env::set_current_dir("thisdoesnotexist").unwrap_err().kind(), ErrorKind::NotFound);
```
Also, you can replace `&Path::new("..")` by `".."` (I missed that when reviewing the PR that landed this).